label_studio_converter/utils.py

```diff
@@ -1,27 +1,24 @@
-import io
 import os
-import xml.etree.ElementTree
-import requests
-import hashlib
+import warnings
+
 import logging
-import urllib
 import numpy as np
 import wave
-import shutil
 import argparse
 
+from pathlib import Path
 from operator import itemgetter
 from PIL import Image
-from urllib.parse import urlparse
 from nltk.tokenize import WhitespaceTokenizer
-from lxml import etree
-from collections import defaultdict
+import label_studio_tools.core.label_config as label_config
```

Comment:
follow the import order:

1. all "import ..." 
2. all "from ... import ...

Also avoid using `label_config` as a name - it clashes with frequently used object name, shouldn't be a module. Instead, import all necessary functions from module explicitly, e.g. from ... import parse_config

Reply:
Order fixed.

> Also avoid using label_config as a name - it clashes with frequently used object name, shouldn't be a module. Instead, import all necessary functions from module explicitly, e.g. from ... import parse_config

This is done because we already have parse_config function in the utils module. As it's a public interface I have added deprecattion warning to it and overrided behavior with label_studio_tools implementation.

Reply:
why can't we simply replace it by by the new one?